src/standalone/Vegetation/Canopy.jl

```diff
@@ -543,10 +543,12 @@ function ClimaLand.make_update_aux(
 
         # Update Rd, An, Vcmax25 (if applicable to model) in place
         Vcmax25 = p.canopy.photosynthesis.Vcmax25
+        SIF = p.canopy.photosynthesis.SIF
         update_photosynthesis!(
             Rd,
             An,
             Vcmax25,
+            SIF,
```

Comment:
Can we make these two things independent? Why does update_photosynthesis! needs SIF?

Reply:
we could, it just currently lives in the photosynthesis name space (stored in canopy.photosynthesis.SIF), we thought it was okay for now with a simple model... 

Do you suggest making a new canopy output / model, stored in canopy.SIF, in a Vegetation/SIF.jl file? (similar to what we have, photosynthesis.jl, radiation.jl, stomatalconductance.jl, etc.)

If we do, it will be a bit more involved and a breaking change, but it will allow to have multiple SIF modules

Reply:
maybe we could do that for a next PR?

Reply:
@kmdeck

Reply:
Yes. I think it should be an independent module. If anything it is a radiative variable and it should live in radiation instead of photosynthesis.

Reply:
Like SIF would be equivalent to APAR. We measure it with a radiometer.